### ISSUE TITLE
DEPR: deprecations log for removed issues

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#13777]
Document removed deprecations, originally indicated in #6581. This reference is in reverse chronological order of release.

## 1.0.0
- [x] #16994 ``inplace`` arg to ``.set_axis()`` (0.21.0) (#27525) 
- [x] #17739 ``.get_value()`` and ``.set_value()`` (0.21.0) (#27377)
- [x] #22264 list-likes IN ``str.cat`` (0.24.0) (#27611) 
- [x] #19339 ``IntervalIndex.from_intervals`` (0.23.0) https://github.com/pandas-dev/pandas/pull/27793
- [x] #29608 support for nested renamers in `aggregate`
- [x] #23592 tm.assert_raises_regex (0.24.0) (#29174)
- [x] #23739 keep_tz=False in DatetimeIndex.to_series (0.24.0) (#29731)
- [x] #19751 factorize(order=) kwarg (0.23.0) (#29732)
- [x] #22672 FrozenNDArray.searchsorted signature (v → value) (0.24.0) (#29732)
- [x] #23585 raise_conflic kwarg replaced by errors in .update() (0.24.0) (#29732)
- [x] #22458 Index.shift n in favor of periods (0.24.0) (#29732)
- [x] #18836 skip_footer in pd.read_excel (0.23.0) (#29721)
- [x] #18902 convert_datetime64 in DataFrame.to_records (0.23.0) (#29721)
- [x] #23053 nthreads arg in read_feather (0.24.0) (#29728)
- [x] #18458 DataFrame/Series.as_matrix (0.23.0) (#29720)
- [x] #18258 Series.from_array (0.23.0) (#29720)
- [x] #18477 .asobject (0.23.0) (#29720)
- [x] #17656 DataFrame.blocks, DataFrame.as_blocks() (0.21.0) (#29720)
- [x] #18529 DataFrame.from_items (0.23.0) (#29720)
- [x] #18627 tseries.plotting.tsplot (0.23.0) (#29726)
- [x] #21400 encoding arg in read_stata (0.24.0) (#29722)
- [x] #18577 broadcast & reduce kwarg in .apply() (0.23.0) (#29730)
- [x] #23917 pd.types.is_period, pd.types.is_datetimetz (0.24.0) (#29744)
- [x] #18800 Series.valid (0.23.0) (#29724)
- [x] #18812 .is_copy (0.23.0) (#29724) 
- [x] #18243 .get_ftype_counts() (0.23.0) (#29724) 
- [x] #20239 .get_duplicates() (0.23.0) (#29724)
- [x] #24203 Series.clip_lower, Series.clip_upper, DataFrame.clip_lower and DataFrame.clip_upper (0.24.0) (#29724)
- [x] #23601 ExtensionArray._formatting_values (0.24.0) (#27774)
- [x] #19239/#26137 SparseSeries, SparseDataFrame (0.25) (#28425)
- [x] #19434 dtype arg in make_block_same_klass (0.23.0) (#29723)
- [x] #19265 fastpath arg in Block (0.23.0) (#29723) 
- [x]  #18217 Index.summary() (0.23.0) (#29819 
- [x] #23635 passing an int for usecols in read_excel (0.24.0) (#29795)
- [x] #20772 .freq setting on a datetimelike index (0.23.0) (#29801) 
- [x] #20730 DatetimeIndex.offset (0.23.0) (#29801)
- [x] #23752 MultiIndex constructor kwarg labels, and .set_lables() deprecated in favor of codes (0.24.0) (#29766)
- [x] #21613 MultiIndex.to_hierarachial (0.24.0) (#29766)
- [x] #20613 pd.concat(...., sort=) (0.23.0) (#29786)
- [x] #21036 floordiv integer ndarray and Timedelta (0.23.0) (#29797)
- [x]  #21896 Series.to_csv signature changes (0.24.0) (#29809)
- [x] #20584 ``Series.rolling.apply``etc for the ``raw`` parameter (0.23.0) (#29829)
- [x] #23110 fastpath keyword in Index constructors (0.24.0) (#29725)
- [x] #23539 allowing datetime64 data passed to TimedeltaIndex (0.24.0) (#29794)
- [x] #23937 allowing timedelta64 data passed to DatetimeIndex (0.24.0) (#29794)
- [x] #26744 DataFrame.ftypes, Series.ftype (0.25.0) (#29895)
- [x] #24050 lib.infer_dtype(...., skipna=None (0.24.0) (#29876)
- [x] #24751 Series.cat.categorical (0.24.0) (#29914)
- [x]  #18164 DTI.weekday_name and .dt.weekday_nam (0.23.0) (#29831)
- [x] #22644 from ``Timestamp.tz_localize``, ``DatetimeIndex.tz_localize``, ``Series.dt.tz_localize`` remove errors keyword, replaced by nonexistent / ambiguous (0.24.0) (#29911)
- [x]  #26839 change Timedelta.resolution to Timedelta.resolution_string (0.25.0) (#29910)
- [X] #18731 listifying tuples for groupby (0.23.0) (#29755)
- [x] #20841 Categorical.take default behaviour with -1 (0.23.0) (#29912)
- [x] #23990 passing dtype alias to DatetimeTZDtype(unit) parameter (0.24.0) (#29927)
- [x] #24114 Categorical where casting to object (0.24.0) (#29913)
- [x] #23396 FrozenNDArray (0.24.0) (#29840)
- [x] #10726 DataFrame.sort_index by keyword (0.17.0) (#29931)
- [x] #17295 reindexing for not-found in .loc (0.21.0) (#29802)
- [x] #17758 non-existant keys in MultiIndex (xref to #17295 for Index) (0.23.0) (#29802)
- [x] #17295 non-existant columns in .to_excel() & non-existant keys in .loc (0.21.0) (#29802)
- [x] #20721 .base, .flags, .stride, .itemsize, .data Series/Index attributes (0.23.0) (#29918)
- [x] #26684 DataFrame & Seriesto_dense/to_sparse (0.25.0) (#29900)
- [x] #27106 Index.dtype_str (0.25.0) (#29900) 
- [x] #27199 Categorical.ravel should return a Categorical and not ndarray (0.25.0) (#29900)
- [x] #27145 .get_dtype_counts (0.25.0) (#29900)
- [x] #24694 .fillna(integer) with timedelta (0.24.0) (#29875)
- [x] #20995 passing multiple axes to DataFrame.dropna (0.23.0) (#29875)
- [x] #24048 Series.nonzero (0.24.0) (#29875)
- [x] #21775 Catetgorical.from_codes accepting float codes (0.24.0) (#29875)
- [x] #23621 passing tz and a datetime.datetime/Timestamp with tzinfo in the Timestamp constructor (0.24.0) (#29929)
- [x] #23919 verify_integrity arg in TimedeltaIndex, DatetimeIndex (0.24.0) (#29930)
- [x] #23919 passing start, end, periods to DatetimeIndex or TimedeltaIndex (0.24.0) (#29930)
- [x] #24157 time_rule kwarg in offsets.generate_range (0.24.0) (#24157)
- [x] #23767 Series.str.partition and Series.str.rpartition, replace pat with sep (0.24.0) (#29986)
- [x] #27106 Series.put (0.25.0) (#29986)
- [x] #27106 Series.real & Series.imag (0.25.0) (#29986)
- [x] #26409 Categorical.get_values, SparseArray.get_values, DataFrame.get_values, Series.get_values, Index.get_values (0.25.0) (#29989)
- [x] #26409 SparseArray.values (0.25.0) (#29989)
- [x] #18307 pandas.tseries.converter.register (???) (#30003)
- [x] #???? pandas.plottting._matplotlib.hist._grouped_plot "figsize" kwarg (???) (#30003)
- [x] #???? PlotAccessor keyword-only arguments (???) (#30003)
- [x] #19980 pandas.plotting._timeseries.tsplot (0.23.0) (#30003)
- [x] #26405 Series/DataFrame.compound (0.25) (#30088)
- [x] #22318 join_axes kwarg in pd.concat (0.25.0) (#30090)
- [x] #???? set_axis signature (0.21.0) (#30089)
- [x] #27198 ufunc.outer(Series) (0.25.0) (#30104)
- [x] #24486 box arg in pd.to_datetime (0.25.0) (#30111)
- [x] #24559 i8data for DTI (0.24.0) (#30115)
- [x] #16955 Series.argmin/argmax (0.21.0) (#30113)
- [x] #22535 integer add/sub with DatetimeIndex, TimedeltaIndex, PeriodIndex, Timestamp, Period (0.24.0) (#30117)
- [x] #23264 Y, M in Timedelta and friends (0.25.0) (#30118)
- [x] #???? Index.contains (0.25.0) (#30103)
- [x] #???? how,fill_method kwargs in resample (??? older than Feb 2016) (#30139)
- [x] #26403 ordered=None in CategoricalDtype in favor of ordered=False (0.25.0) (#29955) 
- [x] #9493 remove deprecated `StataReader.data` method (???) (#30176)
- [x] #17328 remove deprecated "index" keyword from read_stata, StataReader, StataReader.read (???) (#30176)
- [x] #24806 passing non-nano dtype to TimedeltaIndex/DatetimeIndex (0.24.0) (#30199)
- [x] #15113/#26438 .ix (0.20.0 / DeprecationWarning), (0.25.0 FutureWarning) (#27620)
- [x] #27103 pd.to_msgpack and pd.read_msgpack (0.25.0) (#30112)
- [x] #27112 Series.item & Index.item (0.25.0) (#30175)
- [x] #20613 DataFrame.append "sort" keyword default change None to False (0.23.0) (#30251)
- [x] #21614 `Series.ptp (0.24.0) (#30458)
- [x] #21930 ``Series.compress`` (0.24.0) (#30514)

## 0.25.0
- [x] #15601 / #19247``Panel`` (0.20.0) (#27101 et al)
- [x] #15931 ``.agg(..)`` for groupby/rolling/resample with a renaming dict (0.20.0) (depends on #18366 for a proper replacement) (#26399)
- [x] #19269 / #19373 ``pandas.core.categorial`` move (0.23.0) (#25655)
- [x] #16942 ``pd.TimeGrouper`` (0.21.0) (#26477) 
- [x] #16442, #20938 ``ExcelWriter.sheetname`` (0.21.0, 0.23.0) (#26464) 
- [x] #17703 ``pd.TimeGrouper`` (0.21.0) (#26477)
- [x] #16488 ``ExcelWriter.parse_cols`` (0.21.0) (#26522)
- [x] #16970 ``pd.options.html.display`` (0.21.0) (#26540)
- [x] #16915 `convert_objects`(0.21.0), xref #11221, #11173 (initial deprecation) (#26612)
- [x] #17633 ``Series.select``, ``DataFrame.select`` (0.21.0) (#26641)
- [x] #17842 ``reindex_axis`` (0.21.0)
- [x] #17842 mappers for ``rename_axis`` (0.21.0)
- [x] #17982 ``cat.rename_categories`` with a ``Series`` as list-like (0.21.0)
- [x] #17812 ``.from_csv()`` (0.21.0)
- [x] #17820 ``tupleize_cols`` in ``read_csv`` (0.21.0)
- [x] #17877 ``tupleize_cols`` in ``to_csv`` (0.21.0)
- [x] #17774 ``.read_excel(..., parse_cols)`` (0.21.0)
- [x] #17744 ``.where/.mask(..., raise_on_error)``  (0.21.0)
- [x] #17742 ``.astype(...., categories=, ordered=)`` (0.21.0)
- [x] #17691 ``cdate_range`` (0.21.0)
- [x] #17493 ``SeriesGroupBy.nth``, for ``dropna=True`` (0.21.0) (#27168)
- [x] #17352 ``.take(..., convert=)`` param (0.21.0) (#27171)
- [x] #17346 duplicate elements in names for read_csv (0.21.0) (#27175)
- [x] #21359 compare datetime.datetime and datetime.date (0.23.0) (#27109)

## 0.24.0
- [x] #10892 Deprecate `LongPanel/WidePanel` (0.17.0), aliases since < 0.11.0 (#15748, #15802 revert); waiting on statsmodels 0.9.0 release (#18341)
- [x] ``pandas.core.datetools``, deprecated in 0.19.0 (https://github.com/pandas-dev/pandas/commit/3f3839b6cb00a7fbabfa8c0899be69c6ea088b3a) (#19119)
- [x] #14432 ambiguous name in level/column when grouping (#22415)
- [x] #14645 ``Series.repeat(reps)`` (0.20.0) (#22417)
- [x] #14645 ``Index.repeat(n)`` and ``MultiIndex.repeat(n)`` (#22574)
- [x] #15257 the ``as_indexer`` keyword in `str.match()` (0.20.0) (#22626)
- [x] #14645 ``Categorical.searchsorted(v)`` and ``Series.searchsorted(v)`` (#22670)
- [x] #14645 ``TimedeltaIndex.searchsorted(key)``, ``DatetimeIndex.searchsorted(key)``, and ``PeriodIndex.searchsorted(key)`` (#22670)
- [x] #15537 ``pandas.json`` (#22737)
- [x] #14686 ``SparseArray.to_dense(fill)`` (0.20.0) (#22910)
- [x] #14686 ``SparseSeries.to_dense(sparse_only)`` (0.20.0) (#22910)
- [x] #14967 ``.astype(raise_on_error)``  (0.20.0) (#23374)
- [x] #15099 ``Series.sortlevel`` and ``DataFrame.sortlevel`` (0.20.0) (#23375)
- [x] #15358 import of ``concat`` from ``pandas.tools.merge`` (#23376)
- [x] #16005 top-level scatter_matrix and plot_params + plotting methods in pandas.tools.plotting (0.20.0) (#23376)
- [x] #15501 ``.consolidate`` (0.20.0) (#23377)
- [x] #15537 ``pandas.parser``, ``pandas.lib``, ``pandas.tslib`` (0.20.0) (#23378)
- [x] #15538 ``pd.pnow()``, ``pd.match()``, ``pd.groupby()``, ``pd.Expr``, ``pd.Term`` (0.20.0) (#23380)
- [x] #15940 ``pd.get_store()`` (0.20.0) (#23380)
- [x] #16157 ``pandas.computation.expressions.set_use_numexpr()`` (0.20.0) (#23386)
- [x] #16157 ``pandas.types.concat.union_categorical()`` (0.20.0) (#23386)
- [x] #16250 ``shims for util and test`` (0.20.0) (#23386)
- [x] #16189 ``pandas.api.types.is_sequence`` (0.20.0) (#23390)
- [x] #16163 ``pandas.api.types.is_any_int_dtype``, ``is_floating_dtype`` (0.20.0) (#23390)
- [x] #15987 ``.astype(np.datetime64)`` and ``.astype(np.timedelta64)``, unit-less astypes (0.20.0) (#23392)

## 0.23.0
- [x] #11603 `freq`, `how` keywords (0.18.0) (#18601)
- [x] #13706 `pivot_annual` (0.19.0) (#18370)
- [x] #13739 `isleapyear` (0.19.0) (#18370)
- [x] #13358 `pd.ordered_merge` -> `pd.merge_ordered` (0.19.0) (#18459)
- [x] #11603 `freq` from ``.rolling/.expanding`` (#18601)
- [x] #11603 `how` from ``.rolling/.expanding`` (#18668)
- [x] #13735 import errors for io.data and io.wb (0.19.0, but deprecated in 0.17.0) (#18612)
- [x] #14007 `SparseList` (0.19.0) (#18621)
- [x] #13854 `Categorical.from_array` (0.19.0) (#18642)
- [x] #13386 `skip_footer` in `pd.read_csv` (0.19.0) (#18724)
- [x] #13373 `as_recarray` in `pd.read_csv` (0.19.0) (#18804)
- [x] #13360 `compact_ints` and `use_unsigned` in `pd.read_csv` (0.19.0) (#18851)
- [x] #13593 `.freq` in `Timestamp` rather than `.offset` (0.19.0) (#18927)
- [x] #13012 `Series.reshape` and `Categorical.reshape` (0.19.0) (#18954)
- [x] #11603 `pd.rolling_*`, `pd.expanding_*`, `pd.ewm*` (0.18.0); waiting on `dask` release (#18723)
- [x] #13874 `pandas.tseries.frequencies.get_standard_freq` & `to_offset` freqstr kw (0.19.0) (#19023)
- [x] #13776 `Panel4D\Panelnd` (0.19.0) (#19059)
- [x] #19107 ``display.line_width`` and ``display.height``
- [x] ``Categorical.labels`` (#19120)
- [x] ``flavor`` parameter of ``SQLiteDatabase`` (https://github.com/pandas-dev/pandas/commit/8acfad343c88760a6d09fea221996dd50393fa8a) (#19121)
- [x] ``str.extract(expand=None)`` (https://github.com/pandas-dev/pandas/commit/67730ddf46cc16c9c59a3260d37ccaba2603663a) (#19118)
- [x] #16223 ``pandas.tools.hashing``, ``pandas.util.hashing`` (0.20.0) (#19181)
- [x] #13147 `pandas.core.common` dtype introspection functions (0.19.0) https://github.com/pandas-dev/pandas/pull/19769
- [x] resample compat #20554 
- [x] ``axis=None`` param in ``.replace()`` #20789 

## 0.21.0
- [x] #10967 remove `ExcelWriter.has_index_names` (0.17.0) (PR #16522)
- [x] #12882 `Categorical.sort` (0.18.1) (PR #16728)
- [x] #12190 `options.display.mpl_style` (0.18.0) (PR https://github.com/pandas-dev/pandas/pull/16761)
- [x] #12591 `Index.sym_diff` (0.18.1) (PR https://github.com/pandas-dev/pandas/pull/16760)
- [x] #11149 `pd.eval` change `inplace=False` rather than `inplace=None` (0.18.0) (PR #16732) 
- [x] #11834 `get_offset_name` removed (0.18.0) (PR #16863) 
- [x] #2881 ``pd.options.display.line_width`` (PR #16993)
- [x] #3663 ``pd.options.display.height`` (PR #16993)

## 0.20.0
- [x] #10890 remove `TimeSeries` alias (0.17.0) (PR #15098)
- [x] #11135 remove `Series.is_timeseries` (0.17.0) (PR #15098)
- [x] #11308 `pandas.io.ga` (0.17.1) -> (PR #15223)
- [x] #11898 `pd.fama_macbeth`, `pd.ols`, can also remove `statsmodels` as a dep (0.18.0), (PR #15353)
- [x] #10711 `.irow`, `.icol`, `.iget_value` (from 0.11.0, but deprecated in 0.17.0) (PR #15547)
- [x]  #10711 `DataFrame.iterkv` (pretty old, but changed `DeprecationWarning` to `FutureWarning` only in 0.17) (PR #15650)
- [x] #10632 `Categorical.name` (0.17.0) (PR #15654)
- [x] #11121 `generate_bq_schema` remove as no longer needed (0.17.0) (PR #15484)
- [x] #10458 `testing.assert_isinstance` (0.17.0) (PR #15652)
- [x]  #10236`take_last` kw of `duplicated` and `drop_duplicates` (0.17.0) (#15710)
- [x]  #10920`take_last` kw of `Series.nlargest` and `nsmallest` (0.17.0) (#15710)
- [x]  #10792 `take_last` kw of `Groupby.nlargest/nsmallest`(0.17.0) (#15710)
- [x] #10726 `Series.order,sort`, `Index.order`, `DataFrame.sort` (0.17.0) (#15735)
- [x] #12027 back-compat for where with non-strings  in `.select(...., where=)` for `HDFStore` (0.11.0/0.18.0) (#https://github.com/pandas-dev/pandas/pull/15798)
- [x] #10735 `combineAdd` / `combineMult` (0.17.0) (#15805)
- [x] #10748 Deprecate having `np.nan` in `Categorical.categories` (0.17.0) (#15806)

## 0.19.0
- [x] Deprecate colSpace in favor of col_space (no issue, commit 4a5a677) (PR #13857)
- [x] #8227 '+'/'-' for Index set ops (0.15.0) -> https://github.com/pydata/pandas/pull/14127
- [x] #9094 make DTI-DTI subtraction yield a `TimeDeltaIndex` (rather then a set op `.difference()`) (0.16.0) -> #14164 
- [x] #9357,  #3445 remove trellis plot. (0.16.0) (PR #13855)
- [x]  #6910 Change `Panel.shift`'s signature to match `generic.shift()`'s (0.14.0) -> #14041
- [x] `flavor='mysql'` in `to_sql` (0.14.0) -> #13611)
- [x] `tquery`/`uquery` functions (0.14.0) -> #13616
- [x] #11274 `engine` kw in `to_csv()`, from 0.13 (0.17.1), #13419 
- [x] #8376 remove `.levels` from `Categorical` constructor / accessor (0.15.0), #13612 
- [x] #8486 remove `outtype` in `.to_dict()` replaced by `orient` (0.15.0), #13627
- [x] #8227 `diff` for Index set ops (0.15.0), #13669 
- [x] #10951 Legacy offsets (0.17.0) (#13590)
- [x] #9615 remove `pandas.sandbox` (0.16.0), #13670
- [x] #9611, remove ability to set `cat.ordered` directly (0.16.0), #13671 
- [x] #11157 `SparsePanel` (0.17.0), #13778 
- [x] #6919 Remove `copy` keyword from `xs`,`minor_xs`,`major_xs` (4 places) (0.14.0), #13781
- [x] #10085 remove `return_type='series'|'frame'` (replaced by `expand=`) (0.16.1), #13701
- [x] `to_wide` deprecated since 2011 (https://github.com/pydata/pandas/commit/1134c9f6c40398d8eba37c312c88ce294e31c6c5) -> #14039
- [x] boxplot https://github.com/pydata/pandas/pull/7096 (0.14.0) (#12216)
  - [x] change default of `return_type` from None to `'axes'`
  - [x] update return_type section in visualization.rst
## 0.18
- [x] #4950 - `rolling_corr_pairwise()`,  `expanding_corr_pairwise()` replace by `rolling_corr()`, `expanding_corr()` (0.14.0), in #11603 
- [x] #6680, cols was deprecated in DataFrame.to_duplicated and drop_duplicates (0.14.0), #12165 
- [x] #6292, sql functions (#12205)
- [x] #6930 factorize took an `order` argument but didn't do anything with it. (0.14.0) (https://github.com/pydata/pandas/pull/12274)
- [x]  #4892 Float indexers in non-float indexes. (0.14.0) (#12246)
- [x] encoding `items` (in favor of `locs`) in msgpack (#10623) (0.17.0) (#12129)
## 0.17
- [x]  Deprecate colSpace in favor of col_space (no issue, commit 4a5a677c)  (0.8.0 I think) -> Removed #10890
- [x] #2304 Deprecate broadcasting TimeSeries along DataFrame index (0.10.0) -> Removed #10890 
- [x] #5231 `na_last` arg from Series.sort (0.14.0) -> Removed #10726 
- [x] #7088 Deprecate percentile_width in favor of percentiles (0.14.0) ->  Removed in #10881
- [x] #3787 Deprecate load and save in favor of pickle and to_pickle (0.12.0) -> Removed #10892 
- [x]  #4853, #4864 Deprecate timeRule and offset in favor of freq ()  (0.13.0) -> Removed #10892 
- [x]  #4713 Remove kind from read_excel (0.13.0) -> Removed #10892 
- [x]  #4770, #7032 Deprecate infer_types to have no effect (0.13.0) -> Removed #10892 
- [x]  #4645 Deprecate table keyword in HDFStore (0.13.0) -> Removed #10892 
## 0.16
- [x] #5505, DataFrame.pivot_table and crosstab were refactored to use the same arguments as pivot (index and columns) and FutureWarnings added for the old arguments.  Per @jreback, in 0.16, these deprecated arguments should be removed and the docstring updated.  This issue is submitted as a reminder to do so and should be tagged with a 0.156 milestone.
- [x] #6686, cols was deprecated in DataFrame.to_excel and to_csv (rows is not deprecated, don't touch it!)
- [x] #8140, remove `covert_dummies` from `core/reshape` entirely. Deprecated in favor of enhanced `get_dummies`. Deprecated as of 0.15
- [x] #8481, remove `pandas.tools.util.py/value_range`
  ## 0.15
- [x] `DataFrame.delevel` already deprecated in (0.7)

**Comment by jreback** [comment]
@gfyoung if you are looking for something to do :)

I added a couple of old deprecations that were not on the original list......

**Comment by gfyoung** [comment]
I wonder if it is worthwhile to remove `na_last` in `Series.order` (this is the only place where it is used), as the method itself is deprecated and the `na_last` parameter does not get propagated.  Might as well do it all at once when we remove `Series.order`.

**Comment by jreback** [comment]
generally better not to touch a deprecated method partially

**Comment by gfyoung** [comment]
Then we should remove the first unchecked box under the `0.19.0` list then.

**Comment by gfyoung** [comment]
@jorisvandenbossche : Could you check off the two unchecked boxes under the `0.21.0` section?  Those were the PR's that you merged today :smile:

**Comment by topper-123** [comment]
``DataFrame.select`` is missing under 0.21.

**Comment by gfyoung** [comment]
@pandas-dev/pandas-core : Can this be closed?  It looks like all boxes are checked.

**Comment by jorisvandenbossche** [comment]
Yes, this can be closed I think. Also, from now on we are going to tick the boxes in the original issue logging the deprecations, and not moving them to a new issue like this one. The issue to log deprecations in 1.x (removed in 2.0) is https://github.com/pandas-dev/pandas/issues/30228
